spec/integration/resource/api600/synergy/managed_san/delete_spec.rb

```diff
@@ -11,12 +11,10 @@
 
 require 'spec_helper'
 
-klass = OneviewSDK::API500::Synergy::ManagedSAN
+klass = OneviewSDK::API600::Synergy::ManagedSAN
 RSpec.describe klass, integration: true, type: DELETE, sequence: rseq(klass) do
-  let(:current_client) { $client_500_synergy }
+  let(:current_client) { $client_600_synergy }
   let(:san_manager_ip) { $secrets_synergy['san_manager_ip'] }
-  let(:fc_network_class) { OneviewSDK::API500::Synergy::FCNetwork }
```

Comment:
not sure why you have removed these lines? Please make sure if they are working as expected